sapp/ui/issues.py

```diff
@@ -462,26 +476,100 @@ def where_filter(self, filter_instance: Filter) -> "Instance":
             .where_status_is_any_of(filter_instance.statuses)
         )
 
+        if filter_instance.callables is not None and len(filter_instance.callables) > 0:
```

Comment:
Both `None` and empty collections are falsey, so this is more concise
```suggestion
        if filter_instance.callables:
```